--- /etc/xrdp/sesman.ini.modified	2025-04-14 09:31:55.424276460 +0200
+++ /etc/xrdp/sesman.ini	2025-04-14 09:33:53.761837944 +0200
@@ -12,13 +12,13 @@
 ReconnectScript=reconnectwm.sh

 [Security]
-AllowRootLogin=true
+AllowRootLogin=false
 MaxLoginRetry=4
 TerminalServerUsers=tsusers
 TerminalServerAdmins=tsadmins
 ; When AlwaysGroupCheck=false access will be permitted
 ; if the group TerminalServerUsers is not defined.
-AlwaysGroupCheck=false
+AlwaysGroupCheck=true
 ; When RestrictOutboundClipboard=all clipboard from the
 ; server is not pushed to the client.
 ; In addition, you can control text/file/image transfer restrictions
